src/main/java/com/autotune/experimentManager/core/ExperimentTrialHandler.java

```diff
@@ -44,6 +47,19 @@ public class ExperimentTrialHandler {
 
     public ExperimentTrialHandler(ExperimentTrial experimentTrial) {
         this.experimentTrial = experimentTrial;
+        String experimentName = experimentTrial.getExperimentName();
+        String trialNum = String.valueOf(experimentTrial.getTrialInfo().getTrialNum());
+        ConcurrentHashMap<String, HashMap<String, ExperimentTrial>> expTrailMap = EMMapper.getInstance().getExpTrialMap();
```

Comment:
Can you store like this
 ```
         { 
                  "expName1" :  ExperimentTrialObject1 ,
                  "expName2" :  ExperimentTrialObject2 ,
          }    
```
and inside ExperimentTrail object navigate to find if trail number exist. if exist return 400 else 200  . Please move this logic to some class and call this class in CreateExperimentTrial servlet. 

      
        ` this.experimentTrial.getTrialDetails().forEach((tracker, trialDetails) -> {
            trialDetails.getPodContainers().forEach((imageName, podContainer) -> {
                podContainer.getTrialConfigs().forEach((trialNumber, containerConfigData`

Reply:
I'm expecting we will lose the track of the older trial's if we maintain it in a map of exp name and trial object, From my understanding every experiment has multiple trials identified by trial num. If i store exp 1 and trial 0 in the map and when trial 1 gets created the map is updated with trial 1 and details of the trial 0 will be lost (unless we store it in a DB) so for now I would suggest we maintain it in a exp name and list of trials or map of trials (which I have choose to identify a trial object based on exp name and trial number).